backend/getDataFiber.js

```diff
@@ -80,11 +80,12 @@ function getDataFiber(fiber: Object, getOpaqueNode: (fiber: Object) => Object):
       break;
     case HostComponent:
       nodeType = 'Native';
-      name = typeof fiber.type === 'string' ?
-        fiber.type :
-        // Necessary for React Native Fiber if host types are not strings.
-        // https://github.com/facebook/react/pull/9013
-        getDisplayName(fiber.type);
+      name = fiber.type;
+
+      // TODO: we plan to remove this prefix anyway.
```

Comment:
Feel free to tag me by name in this TODO if you'd like. It's on my TODO list to remove the prefix in RNF anyway.